reactComponents/src/searchbar/tokenSearch/SearchFiltersNetworkSelectors.tsx

```diff
@@ -1,34 +1,54 @@
-import React from "react"
+import React, { useContext } from "react"
 import { useDispatch, useSelector } from 'react-redux';
 import { omitBy } from "lodash"
 import { setNetworkMap, setNetworkMapAll, setExchangeMapAll } from "../redux/tokenSearchSlice"
 import { networkNames } from "./helpers/config";
-import { Chip } from "../Components/Chip";
+import { Chip } from "./Chip"
 import {RootState} from "../redux/store";
-
+import TokenSearchContext from '../Context/TokenSearch';
 
 export const FilterNetworkAll = () => {
   const dispatch = useDispatch();
+  const renderProps = useContext(TokenSearchContext);
+
   const { exchangeMap, networkMap } = useSelector((state:RootState) => state);
   const networkAll = Object.values(omitBy(networkMap, b => !b)).length === 0;
   const exchangeNamesActive = Object.keys(omitBy(exchangeMap, b => !b));
-
+  
+  const { customAllChip } = renderProps
+  const styles = {
+    fontSize: customAllChip?.fontSize || '7px',
+    fontWeight: customAllChip?.fontWeight || '500',
+    borderRadius: customAllChip?.borderRadius || "4px",
+    backgroundColor: customAllChip?.backgroundColor || "#474F5C",
+    border: customAllChip?.border || "0",
+    padding: customAllChip?.padding || "3px 2px",
+    margin: customAllChip?.margin || "0",
+    defaultColor: customAllChip?.defaultColor || "#7A808A",   
+    width: customAllChip?.width || "auto",
+    height: customAllChip?.height || "auto",
+    textAlign: customAllChip?.textAlign || "center" ,
+    textTransform: customAllChip?.textTransform || "inherit",
+    gridTemplateColumns: customAllChip?.gridTemplateColumns || "40px",
+    justifySelf: customAllChip?.justifySelf || "center",    
+  }
 
   // RENDERING.
   return <Chip
     name={'AllNetworks'}
-    label={'All'}
+    label={'Select All'}
     checked={networkAll}
+    styles={styles}
```

Comment:
would this be better named as styleOverrides?

Reply:
I agree, not problem.